lib/vrl/web-playground/public/index.html

```diff
@@ -88,83 +88,150 @@ <h1>[draft] VRL playground wasm</h1>
 
 
 .timestamp = to_unix_timestamp(ts)`,
-        language: 'python',
-        theme: 'vs-light',
-        minimap: { enabled: false }
-      });
-
-      window.eventEditor = monaco.editor.create(document.getElementById('container-event'), {
-        value: `{
+          language: "coffeescript",
+          theme: "vs-light",
+          // minimap: { enabled: false },
+          automaticLayout: true,
+        });
+
+        window.eventEditor = monaco.editor.create(document.getElementById("container-event"), {
+          value: `{
   "message": "<102>1 2020-12-22T15:22:31.111Z vector-user.biz su 2666 ID389 - Something went wrong"
 }`,
-        language: 'json',
-        theme: 'vs-light',
-        minimap: { enabled: false }
+          language: "json",
+          theme: "vs-light",
+          // minimap: { enabled: false },
+          automaticLayout: true,
+        });
+
+        window.outputEditor = monaco.editor.create(document.getElementById("container-output"), {
+          language: "json",
+          theme: "vs-light",
+          // minimap: { enabled: false },
+          automaticLayout: true,
+        });
+        const queryString = window.location.search;
+        if (queryString.length != 0) {
+          const urlParams = new URLSearchParams(queryString);
+          const stateParam = decodeURIComponent(urlParams.get("state"));
+
+          try {
```

Comment:
we needed to add a try catch statement for the case when the user inputs an invalid url and run_vrl doesn't run, previously nothing would show up in the output box but now we get an error message detailing that might be the issue.

Reply:
This was something I noticed previously. It seems to behave better now